### ISSUE TITLE
Idea: Allow Toggling Defaults

### ISSUE THREAD

**Issue by hryanjones** [vega/editor#137]
I'm quite new to Vega-lite and it's editor. The inline annotations on possible values are very nice for exploring what's possible without referring to documentation, but finding the right keyword to specify something can be a little difficult.

My idea is to have a mode where the defaults could be toggled on (maybe they'd show up as gray text), and when they're edited they'd become part of the chart specification.

This doesn't sound like a particularly easy request, so of course feel free to ignore it, but I thought I'd share the idea.

Best,
Ryan

**Comment by domoritz** [comment]
For Vega-Lite, you can look at the generated Vega to see what is happening. I like the idea but it's indeed very complicated to implement and I doubt we will add this.

**Comment by hryanjones** [comment]
Yes, the tooltip is very helpful, but knowing the keyword to get to it was the harder part.

Thanks for responses on this :)
